dali/operators/coord/coord_flip.cc

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/coord/coord_flip.h"
+
+namespace dali {
+
+DALI_SCHEMA(CoordFlip)
+    .DocStr(
+        R"code(Transforms normalized coordinates (range [0.0, 1.0]) so that they map to the same place after
```

Comment:
How about adding also a version for not normalized coordinates? We have tensor shape, so simple switch in API makes do

Reply:
done with the new patch